src/Microsoft.AspNet.Mvc.Localization/HtmlLocalizer.cs

```diff
@@ -153,120 +141,9 @@ public virtual LocalizedHtmlString Html(string key, params object[] arguments)
         /// </summary>
         /// <param name="result">The <see cref="LocalizedString"/>.</param>
         protected virtual LocalizedHtmlString ToHtmlString(LocalizedString result) =>
-            new LocalizedHtmlString(result.Name, result.Value, result.ResourceNotFound);
-
-        /// <summary>
-        /// Encodes the arguments based on the object type.
-        /// </summary>
-        /// <param name="resourceString">The resourceString whose arguments need to be encoded.</param>
-        /// <param name="arguments">The array of objects to encode.</param>
-        /// <returns>The string with encoded arguments.</returns>
-        protected virtual string EncodeArguments(string resourceString, object[] arguments)
```

Comment:
We may need to bring this back for perf reasons if localization requires it. Also, object-pooling for stringbuilders...